CONTRIBUTING.md

```diff
@@ -0,0 +1,174 @@
+# Contributor guidelines
+
+# Contributing
+1. Fork it (https://github.com/HewlettPackard/py-spiffe/fork)
+2. Clone your forked repository (git clone git@github.com:<your_github_account>/py-spiffe.git)
+3. Create your feature branch (git checkout -b feature/fooBar)
+4. Commit your changes (git commit -m 'Add some fooBar' --signoff)
+5. Push to the branch (git push origin feature/fooBar)
+6. Create a new Pull Request
+
```

Comment:
Suggestion: I would add `To keep track of the changes on the upstream repository: git remote add upstream https://github.com/HewlettPackard/py-spiffe` and then `git fetch --all` to fetch all changes on the remotes, and finally `git rebase upstream/master`. Maybe this can be on an extra section if you consider it useful.

Reply:
Thanks Max for the comments.
I personally consider this out of scope for these guidelines.